app/test/frontend/handlers_test_utils.dart

```diff
@@ -202,12 +202,31 @@ class TemplateMock implements TemplateService {
   }
 
   @override
+  String renderNewPkgShowPage(
```

Comment:
Maybe we should be consistent with what we use `new` or `experimental` or `v2`. I would just go for `v2` and  only use `experimental` in the URL, so users know that this is not stable yet.

Reply:
Renamed them with a V2 postfix.